jenkins-pipelines/features-per-partition-limit.jenkinsfile

```diff
@@ -6,7 +6,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 longevityPipeline(
```

Comment:
I think this file isn't related to this PR, is it?

Reply:
I see now that it covers two features (not only the one in the title)

Reply:
I reused some code from per-partition limit test - so I moved it to different file&class.